primitives/src/locktime/absolute.rs

```diff
@@ -405,12 +405,25 @@ mod tests {
 
     #[test]
     fn display_and_alternate() {
+        let lock_by_height = LockTime::from_height(741_521).expect("valid height");
+        let lock_by_time = LockTime::from_time(1_653_195_600).expect("May 22nd 2022, 5am UTC");
+
+        assert_eq!(format!("{}", lock_by_height), "741521");
+        assert!(!format!("{:#}", lock_by_height).is_empty());
+        assert!(!format!("{:?}", lock_by_height).is_empty());
+
+        assert_eq!(format!("{}", lock_by_time), "1653195600");
+        assert!(!format!("{:#}", lock_by_time).is_empty());
```

Comment:
Why is this only checking for non-empty?

Reply:
I didn't want to put the string in the check in case it changed in future.  I have now changed it to check the string is as expected.

Reply:
Note that it's `Display` impl (no `?` there), so those should stay stable. On the other hand `Debug` impl may change but should not be empty.